Sources/Annotations/MKMapAnnotationView.swift

```diff
@@ -16,31 +16,51 @@ class MKMapAnnotationView<Content: View>: MKAnnotationView {
 
     private var controller: NativeHostingController<Content>?
 
-    // MARK: Methods
+    // MARK: Computed Properties
 
-    func setup(for mapAnnotation: ViewMapAnnotation<Content>) {
-        annotation = mapAnnotation.annotation
```

Comment:
Revert  `annotation` and `clusteringIdentifier` assignment to make clustering feature work